apm-protocol/apm-ui-protocol/src/main/resources/ui-graphql/overview-layer.graphqls

```diff
@@ -0,0 +1,37 @@
+# Query the cluster brief based on the given duration
+type ClusterBrief {
+    numOfApplication: Int
+    numOfService: Int
+    numOfDatabase: Int
+    numOfCache: Int
+    numOfMQ: Int
+}
+
+# Query the trend of alarm rate based on the given duration
+type AlarmTrend {
+    numOfAlarmRate: [Int]!
+}
+
+# Query all conjectural applications based on the given duration
+# All applications here are not installed agent.
+type ConjecturalAppBrief {
+    apps: [ConjecturalApp!]
+}
+
+# The basic info of the conjectural application,
+# includes the type and num of same type application
+type ConjecturalApp {
+    # The display name of the application
+    # e.g. MySQL, RocketMQ, Kafka, Nginx
+    name: String!
+    num: Int!
+}
+
+extend type Query {
+    getClusterTopology(duration: Duration!): Topology
```

Comment:
Topology changed to ApplicationTopology. Topology to be a abstract object.

Reply:
@peng-yongsheng I prefer to keep the `Topology`. The inheritance happens in Node, rather than Topology type.

Reply:
I agree with @wu-sheng